lib/galaxy/datatypes/data.py

```diff
@@ -369,7 +369,6 @@ def display_data(self, trans, data, preview=False, filename=None, to_ext=None, *
             if os.path.exists(file_path):
                 if os.path.isdir(file_path):
                     tmp_fh = tempfile.NamedTemporaryFile(delete=False)
-                    tmp_file_name = tmp_fh.name
```

Comment:
This was due to a bad merge elsewhere in dev, it shouldn't be deleted. It'll fix the failing tests.

Reply:
ah gotcha, reverted it, thanks :)